### ISSUE TITLE
automatic grocery shopping

### ISSUE THREAD

**Issue by mobeets** [hahayes/hahayes-projects#31]
okay, implementing this will have to wait until disposable rfid tags with states exists, or something like it.

anyway, what i want to do is, when something in my kitchen is nearly empty, e.g., a container of olive oil, i want to press a little button on it that makes a light turn red, and then amazon or someone will automatically order it for me and have it delivered.

i'm thinking that this could work by having disposable stickers with buttons on them, and when you come home from grocery shopping you put them on everything, so that when they become empty you just press them to say, "this is empty." the obvious problem is programming what the product is onto the sticker, which i assume could be done somehow by holding a phone up to one and it takes a photo or something to figure out what it is.

(also, i don't know if you've seen these in melbourne, but we have a grocery store that has these [parking lot sensors](http://www.nedapidentification.com/solutions/cases/the-parking-space-race.html), so at each turn in the parking garage you know how many spots are in that row.)

**Comment by silky** [comment]
could you just take a photo of the empty thing; and then maybe use some of the barcode software stuff to gather that up as a list for next time? or automatically order it or whatever?

**Comment by mobeets** [comment]
true, i guess i was thinking you'd take a photo when you first got it, but if you're gonna take a photo you might as well take it when it's empty. it just seemed fun to press a button and see a light! haha

**Comment by silky** [comment]
hahaha

yeah, i agree it _wouldn't_ be fun to have to take a photo of all the food you want re-ordered. it also doesn't work on food that you also eat or dispose of the container; i.e. apples/bananas,etc.
